WhisperAPI/WhisperAPI/Services/MLAPI/Facets/IFacetValues.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace WhisperAPI.Services.MLAPI.Facets
+{
+    public interface IFacetValues
+    {
+        List<Models.MLAPI.FacetValues> GetFacetValues(IEnumerable<string> facetsName);
```

Comment:
I personality don't like it when class are not import in the beginning of the file

Reply:
I can't because another class has the same name, I tried it and it does'nt want because of that

Reply:
I tried to use this: 
using FacetValues = WhisperAPI.Models.MLAPI.FacetValues;
and Use FacetValues after but it does not work